sql/src/main/java/org/apache/druid/sql/avatica/DruidMeta.java

```diff
@@ -63,7 +64,19 @@
 
 public class DruidMeta extends MetaImpl
 {
-  private static final Logger log = new Logger(DruidMeta.class);
+  public static <T extends Throwable> T logFailure(T error, String message, Object... format)
+  {
+    LOG.error(message, format);
+    return error;
+  }
+
+  public static <T extends Throwable> T logFailure(T error)
+  {
+    LOG.noStackTrace().error(error, error.getMessage());
```

Comment:
why this function does not log stack trace?

Reply:
Hmm, I guess I was trying to match the behavior here when logging errors in `QueryLifecycle`, but maybe these are worth having the stack trace handy since I guess they are only happening outside the query lifecycle so maybe more interesting. I'm going to change it I think